libs/barista-components/filter-field/src/filter-field-default-data-source.ts

```diff
@@ -20,15 +20,15 @@ import { isDefined, isObject } from '@dynatrace/barista-components/core';
 import { DtFilterFieldValidator } from './filter-field-validation';
 import { DtFilterFieldDataSource } from './filter-field-data-source';
 import {
-  DtNodeDef,
   dtAutocompleteDef,
-  isDtGroupDef,
-  isDtAutocompleteDef,
-  dtOptionDef,
-  dtGroupDef,
   dtFreeTextDef,
-  dtRangeDef,
+  dtGroupDef,
   dtMultiSelectDef,
+  DtNodeDef,
+  dtOptionDef,
+  dtRangeDef,
+  isDtAutocompleteDef,
+  isDtGroupDef,
```

Comment:
Nitpick.
If it is not too much trouble, could you change these back. There is no change here other than the automatic import sorter of your IDE.